test/e2e/e2e.go

```diff
@@ -87,27 +99,71 @@ var csiTestSuites = []func() testsuites.TestSuite{
 	testsuites.InitMultiVolumeTestSuite,
 }
 
-var _ = ginkgo.BeforeSuite(func() {
-	ginkgo.By(fmt.Sprintf("Creating EFS filesystem in region %q for cluster %q", Region, ClusterName))
-	if Region == "" || ClusterName == "" {
-		ginkgo.Fail("failed to create EFS filesystem: both Region and ClusterName must be non-empty")
+var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
+	// Validate parameters
```

Comment:
For my notes:

>SynchronizedBeforeSuite blocks are primarily meant to solve the problem of setting up singleton external resources shared across nodes when running tests in parallel. For example, say you have a shared database that you can only start one instance of that must be used in your tests. When running in parallel, only one node should set up the database and all other nodes should wait until that node is done before running.

>SynchronizedBeforeSuite accomplishes this by taking *two* function arguments. The first is only run on parallel node #1. The second is run on all nodes, but *only* after the first function completes successfully. Ginkgo also makes it possible to send data from the first function (on Node 1) to the second function (on all the other nodes).

https://godoc.org/github.com/onsi/ginkgo#SynchronizedBeforeSuite